data/rapid_config.yaml

```diff
@@ -0,0 +1,10 @@
+
+# Rapid configuration-  for customizing runtime behavior. 
+# The use case for this file is to enable/disable features without
+# needing to open/edit/rebuild the code. 
+rapid_config: 
+  # This setting styles any features in the map with a special shadow effect if 
+  # and only if they were originally presented to the user as an to be AI 
+  # features
+  style_previous_ai_features:
```

Comment:
does this only apply to FB AI roads (with `source=digitalglobe` tag)? if so, let's call it `style_fb_ai_roads` to be more specific